### ISSUE TITLE
how omit dangerouslySetInnerHTML

### ISSUE THREAD

**Issue by bjatwork** [gatsbyjs/gatsby#14901]
i have to use dangerouslySetInnerHTML for almost every field returned from graphql, there's a way to define which fields are automatically transform to omit using dangerouslySetInnerHTML in comontents

**Comment by wardpeet** [comment]
could you rephrase this issue a little bit? If the graphql is returning html you sadly have to do the `dangerouslySetInnerHTML` manually.

You can create a html component for this

```js
export default function HTML(props) {
  return <div dangerouslySetInnerHTML={{__html: props.html }} />
}
```

**Comment by bjatwork** [comment]
ok thanks, it's hard to get used to use for everything components not function.

**Comment by wardpeet** [comment]
Thanks so much for opening this issue! As stated, it seems like we’ve provided enough information to make an informed decision.

If this is not the case, or if we can help further--please don’t hesitate to reach out or comment on this issue, and we’d love to take another look.

Thanks for using Gatsby 💪
